data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,471 @@ mission "Unfettered returning home"
+	to complete
+		not "unfettered: fled with the relic"
+		has "unfettered: beaten the bugs"
+		has "flagship model: Anomalocaris"
+		or
+			"old reputation: Hai (Unfettered)" > -2000
+			has "unfettered: beaten extra bugs"
+	on complete
+		clear "unfettered: beaten the bugs"
+		clear "unfettered: beaten extra bugs"
+		clear "old reputation: Hai (Unfettered)"
+		log "Managed to use an old Anomalocaris to beat some Unfettered challengers, earning respect with the Unfettered Hai so long as the ship remains in your possession."
+		conversation
+			`When you land in one piece, Mera approaches, looking quite surprised. "You really won! At least it was close, right? Or did the captains make stories to justify their incompetence already like the cowards they are?"`
+			choice
+				`	"It was fun."`
+					goto next
+				`	"What are you talking about? This was easy!"`
+			`	"Don't get overconfident, Captain," she muses. I am sure you know what happens to such people.`
+			label next
+			`	"Well, congratulations, I did not think you had it in you. I will give you authorization to land, on the condition that you keep this relic from our past in one piece. Remember, your reputation with us is solely due to to that ship. If you somehow lose it because of incompetence on your own end, it would look pretty bad to us!"`
+			`	Mera then walks off as she begins reviewing data from your fight on a small device. For now, it seems that the Unfettered Hai have some sort of respect for you, so long as you have the Anomalocaris.`
+
+
+mission "Undeserved Anomalocaris"
+	minor
+	invisible
+	landing
+	source
+		attributes "unfettered"
+	to offer
+		has "ship model: Anomalocaris"
+		"reputation: Hai (Unfettered)" <= -1000
+		not "Hai Legacy 2: active"
+	on offer
+		conversation
+			`You land on this Unfettered world in possession of the Anomalocaris, and, as you do, countless Unfettered head to your ship, each armed as they always are.`
+			`	Except, this time, they seem even more intimidating than usual... They must want the ship.`
+			`	One of them sends you a private message. "You do not deserve to fly this relic of old, and you missed your opportunity for a glorious battle to earn that right!"`
+				to display
+					not "Hai Legacy 3: offered"
+			`	"We will give you another ship so you can leave, but you must surrender this one, and the savage Korath's weapons, if you did not already rob us of those," one of them explains to you in a menacing tone.`
+			`	Another one steps forward, "We will allow to keep the outfits, provided you give it to us on Darkcloak," they say in a somewhat diplomatic tone.`
+				to display
+					not "flagship planet: Darkcloak"
+			`	Another one steps forward, "Since you so kindly came to Darkcloak to deliver the ship to us, we will allow you to buy back any outfits you may still desire, as a courtesy," they say in a somewhat diplomatic tone.`
```

Comment:
```suggestion
			`	Another one steps forward, "Since you so kindly came to Darkcloak to deliver the ship to us, we will allow you to buy back any outfits you may still desire, as a courtesy."`
```